### PR TITLE
Classes/lecture2 tasmia aqila

### PR DESCRIPTION
**Pull request by tasmiaorni**
Homework of Tasmia Aqila

### CODE REVIEW

File: src/main/java/co/getmehired/getmehired/model/Talent.java [hunk 1]
```diff
@@ -7,35 +7,265 @@
...
+	
+	public void setAddress(String address) {
+		this.address = address;
+	}
+	
+	public String getCalendlyUrl() {
+		if(this.calendlyUrl!=null) {
+			return calendlyUrl;
+		}
+		return "calendlyUrl not found";
```
Comment (getmehired) [inline src/main/java/co/getmehired/getmehired/model/Talent.java]: getter and setter should not have any logic. Their task is to only get and set data. The logic check should be done in controller.

File: src/main/java/co/getmehired/getmehired/rest/TalentResource.java [hunk 2]
```diff
@@ -1,29 +1,69 @@
...
+			System.out.println(talent.getGraduationDatestr());
+			System.out.println(talent.getJobStartDatestr());
+			//System.out.println("Invalid Date");
+			
 		}  
+	
+      return talent;
+}
+	
+	@PutMapping("/api/talents/{")
```
Comment (getmehired) [inline src/main/java/co/getmehired/getmehired/rest/TalentResource.java]: what is this '{' here? before checking in your code always run 'mvnw clea install' to make sure code is building successfully.

File: src/main/java/co/getmehired/getmehired/rest/TalentResource.java [hunk 3]
```diff
@@ -1,29 +1,69 @@
...
+	public UpdateInfo updateinfo(@RequestBody Talent talent) {
+		UpdateInfo new_info=new UpdateInfo();
+		new_info.setName(talent.getName());
+		new_info.setSsnNumber(talent.getSsnNumber());
 		
-		return talent;
+		return new_info;
 	}
+	
+  @RequestMapping(value = "/api/talents",method = RequestMethod.DELETE)
```
Comment (getmehired) [inline src/main/java/co/getmehired/getmehired/rest/TalentResource.java:58]: this good. Also you can use @DeleteMapping.

**Review by getmehired — CHANGES_REQUESTED** [review]

File: src/main/java/co/getmehired/getmehired/model/Talent.java (hunk 1 above)
Comment (tasmiaorni) [reply src/main/java/co/getmehired/getmehired/model/Talent.java]: I solved this

File: src/main/java/co/getmehired/getmehired/rest/TalentResource.java (hunk 2 above)
Comment (tasmiaorni) [reply src/main/java/co/getmehired/getmehired/rest/TalentResource.java]: This was typo. solved this.

File: src/main/java/co/getmehired/getmehired/rest/TalentResource.java (hunk 3 above)
Comment (tasmiaorni) [reply src/main/java/co/getmehired/getmehired/rest/TalentResource.java:58]: ok bhaia. I just wanted to keep two different ways that we can use.

**Review by getmehired — APPROVED** [review]

**Pull request closed without merging**